UnicodeJsps/src/main/java/org/unicode/jsp/CachedProps.java

```diff
@@ -43,7 +44,9 @@ public class CachedProps {
     final BiMultimap<String, String> nameToAliases = new BiMultimap<String, String>(null, null);
     final Map<String, BiMultimap<String, String>> nameToValueToAliases = new LinkedHashMap();
 
-    static CachedProps CACHED_PROPS = getInstance(VersionInfo.getInstance(14));
+    static String jspVersion = IS_BETA ? Settings.lastVersion : Settings.latestVersion;
+    static VersionInfo jspVersionInfo = VersionInfo.getInstance(jspVersion);
```

Comment:
Discussion: As long as we are serving ICU data, we should use `jspVersionInfo = UCharacter.getUnicodeVersion();`